Library/MeterImage.cpp

```diff
@@ -170,6 +182,15 @@ bool MeterImage::Update()
 			{
 				if (m_ImageName.empty())
 				{
+					auto plugin_measure = dynamic_cast<MeasurePlugin *>(m_Measures[0]);
+					if (plugin_measure != nullptr)
+					{
+						if (LoadImageFromPluginMeasure(plugin_measure))
+						{
+							CalcImageDimensions();

```

Comment:
I think it would be better to move this line to `LoadImageFromPluginMeasure` function

Reply:
I'll include this change into a PR that adds configuration function into rainmeter API.